recipe/meta.yaml

```diff
@@ -25,7 +25,6 @@ requirements:
   build:
     - patch     # [not win]
     - m2-patch  # [win]
-
   host:
     - python
     - setuptools >=42
```

Comment:
Could you use `ninja-base` instead of `ninja`?

Reply:
There is no impact here as meson has python as run dependency anyway.